licenses/cla-individual.md

```diff
@@ -222,3 +222,5 @@ Andreas Abeck, @andreasabeck 2015/03/18
 Hong-Yi Dai, @reflectionalist 2015/04/04
 
 heroboy, @heroboy 2015/04/19
+
+vouko, @vouko 2015/05/02
```

Comment:
I am only willing (and competent, at least I think) to help about the documentation. I don't have (serious) coding skills.
However, I am not interested in recording forever my real name on the Internet. This is a general attitude. I like my privacy. 

In fact, perhaps I don't really need to sign this while still being able to contribute... ??